plugins/goodreads.koplugin/doublekeyvaluepage.lua

```diff
@@ -0,0 +1,318 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
```

Comment:
This widget depends on gooreaderspi, so it should be part of the plugin. Do intend to make it a generic widget?